analyses/tests/test_psp_fitting.py

```diff
@@ -0,0 +1,118 @@
+from multipatch_analysis.connection_detection import create_all_fit_param_combos, fit_psp
+import os
+import numpy as np
+from pprint import pprint
+import json
+import neuroanalysis.data
+
+test_data_dir = os.path.join(os.path.dirname(neuroanalysis.__file__), '..', 'test_data', 'test_psp_fit')                                                                                                                                      
+
+def test_create_all_fit_param_combos():
```

Comment:
move this to multipatch_analysis/tests/test_connectino_detection

Reply:
done.